docs/source/data.rst

```diff
@@ -14,23 +14,20 @@ An example of this organization is shown below:
     └── dataset_description.json
     └── participants.tsv
     └── sub-01
+    └── sub-02
+    └── sub-03
         └── anat
-            └── sub-siteX01_T1w_reg.nii.gz
-            └── sub-siteX01_T1w_reg.json
-            └── sub-siteX01_T2w_reg.nii.gz
-            └── sub-siteX01_T2w_reg.json
-            └── sub-siteX01_acq-MTon_MTS_reg.nii.gz
-            └── sub-siteX01_acq-MTon_MTS_reg.json
-            └── sub-siteX01_acq-MToff_MTS_reg.nii.gz
-            └── sub-siteX01_acq-MToff_MTS_reg.json
-            └── sub-siteX01_acq-T1w_MTS.nii.gz
-            └── sub-siteX01_acq-T1w_MTS.json
-            └── sub-siteX01_T2star_reg.nii.gz
-            └── sub-siteX01_T2star_reg.json
+            └── sub-03_T1w.nii.gz  <-- MRI image in NIfTI format
+            └── sub-03_T1w.json  <---- Metadata including image parameters, MRI vendor, etc.
+            └── sub-03_T2w.nii.gz
+            └── sub-03_T2w.json
     └── derivatives
         └── labels
-            └── sub-siteX01
+            └── sub-03
```

Comment:
Do each of the datasets need to have a labels folder? If so, it might be an idea to include `sub-01` and `sub-02` here as well?

Reply:
> Do each of the datasets need to have a labels folder? If so, it might be an idea to include `sub-01` and `sub-02` here as well?

sub-01 and sub-02 are supposed to have one as well. It is implied here (for the sake of clarity).